pkg/controller/replication_controller.go

```diff
@@ -67,6 +67,11 @@ const (
 	// Realistic value of the burstReplica field for the replication manager based off
 	// performance requirements for kubernetes 1.0.
 	BurstReplicas = 500
+
+	// When controllers notice that the replication manager isn't ready, it sleeps this
+	// long before queuing itself for a sync once again. This is so rcs give the rc manager
+	// a chance to initialize itself instead of hogging cpu in a tight loop.
+	ManagerReadinessPollPeriod = 100 * time.Millisecond
```

Comment:
// We must avoid counting pods until the pod store has synced. If it hasn't synced, to avoid a hot loop, we'll wait this long between checks.

Also, "manager readiness" is too vague. Suggest a term that explicitly only covers the pod store. It can be expanded later if need be.